processor/src/main/java/org/mapstruct/ap/internal/gem/ComponentModelGem.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.internal.gem;
+
+/**
+ * Gem for the class {@link org.mapstruct.ComponentModel}
+ *
+ * @author Tomas Poledny
+ */
+public final class ComponentModelGem {
```

Comment:
This class is not needed anymore, right? Since you added a gem in the `MappingContsantsGem`.

Reply:
Oh my bad. It is removed.